Makefile

```diff
@@ -153,7 +153,10 @@ all: build
 init:
 	./hack/setup_env.sh assisted_service
 
-ci-lint:
+vendor-diff:
+	go mod vendor && git diff --exit-code vendor
+
+ci-lint: vendor-diff
```

Comment:
``ci-lint`` is rather unused, but you can add vendor generation to CI right in this PR
You can have in ``hack/Makefile``:
```
generate-vendor:
    go mod vendor

generate: generate-vendor generate-from-swagger generate-events generate-mocks generate-configuration
```

Reply:
>ci-lint is rather unused

Then where do we validate tehat generation doesn't result in diff?

Reply:
https://github.com/openshift/release/blob/d50e1472bb34baaee6f240ddafc9a0b467d250f5/ci-operator/step-registry/assisted/verify-generated-code/assisted-verify-generated-code-commands.sh#L14

Reply:
Maybe we can do diff of git-diffs, but it's rather ugly